### ISSUE TITLE
Signing profile?

### ISSUE THREAD

**Issue by spendres** [corbado/flutter-passkeys#27]
I tried to use the passkeys/passkeys/example with an iPhone14 simulator, but you may need to provide an xcode signing profile to allow simulators to use the corbado domains. This is what I get when I try to signup:
A Dart VM Service on iPhone 14 is available at: http://127.0.0.1:58211/8JIzFRpqDNo=/
The Flutter DevTools debugger and profiler on iPhone 14 is available at:
http://127.0.0.1:9101?uri=http://127.0.0.1:58211/8JIzFRpqDNo=/
flutter: PlatformException(domain-not-associated, The operation couldn’t be completed. Application with identifier YEFSQLA92K.com.corbado.passkeys.pub is not associated with domain pro-6244024196016258271.frontendapi.corbado.io, , null)

flutter: CorbadoPasskeyBackend: pro-6244024196016258271

**Comment by vincentdelitz** [comment]
@spendres we're on it and keep you posted

**Comment by incorbador** [comment]
Hi @spendres. We fixed the example on main now. Can you please pull main and run the example again?

Our approach for the example is quite simple right now. We configured "project.pbxproj" with a static DEVELOPMENT_TEAM ("0000000000") and PRODUCT_BUNDLE_IDENTIFIER ("com.corbado.passkeys.pub"). We need to know these values in advance as we need to whitelist them at the relying party server that is used in the example. 

On a simulator this is not a problem though as a real signing profile is only needed to test the app on physical devices. Please don't update the DEVELOPMENT_TEAM or the PRODUCT_BUNDLE_IDENTIFIER when you open the example in XCode.

Feel free to add more comments here if you have further questions or issues connected to this topic :)

**Comment by spendres** [comment]
To make this example work:
1. `xed .` within iOS directory... to open XCode
2. Within XCode, use the File → Open Simulator menu option(I used iPhone14). 
3. `flutter run` 

Otherwise `flutter run` overwrites project.pbxproj as it tried to build the app for and run the project on my device. I had to `git restore` the modified files, then open a simulator before executing `flutter run`. Then the demo worked on the simulator.  

Thanks for updating this example.
